tests/sentry/workflow_engine/migration_helpers/test_migrate_alert_rule.py

```diff
@@ -417,3 +420,92 @@ def test_create_metric_alert_trigger_action_no_type(self, mock_logger):
                 "alert_rule_trigger_action_id": self.alert_rule_trigger_action_integration.id,
             },
         )
+
+    def test_update_metric_alert(self):
+        migrate_alert_rule(self.metric_alert, self.rpc_user)
```

Comment:
it might be nice to assert on the pre-update values to make it easier to see that it's changed